prefab/README.md

```diff
@@ -1,27 +1,27 @@
 Prefab Samples
 ==============
-The samples in this directory demonstrate how to create and consume C/C++
-dependencies with the Android Gradle Plugin. Dependencies are packaged within
+The samples in this directory demonstrate how to create and consume [C/C++
+dependencies] with the Android Gradle Plugin. Dependencies are packaged within
 AARs using the [Prefab] package format. The Android Gradle Plugin natively
 supports producing and consuming these, so you do not need to worry about the
 details of the packaging format.
 
 The individual samples will explain in more detail, but the typical workflow for
 a consumer of a native dependency is:
 
-1. Enable the prefab build feature in your build.gradle file.
-1. Add the dependency to the dependencies block of your build.gradle file.
-1. Import the package in your CMakeLists.txt or Android.mk file.
+1. Enable the prefab build feature in your `build.gradle` file.
+1. Add the dependency to the dependencies block of your `build.gradle` file.
+1. Import the package in your `CMakeLists.txt` or `Android.mk` file.
 1. Link the dependencies to your libraries.
 
 Linking the dependency to your library will automatically make the headers
-available, link the required libraries, and include the dependencies you use in
-your APK or AAR.
+available, link the required libraries, and pack the dependencies you use into
```

Comment:
previous wording "include the dependencies you use in" was better

Reply:
changed